message_app/model.py

```diff
@@ -2,8 +2,25 @@
 from __future__ import annotations
 from message_app.db.db import DB as db
 import json
+import time
 from typing import Union, List
 
+# TODO: Optimize all models by removing the duplicated codes.
+# All the models will be REFACTORED in the future for more functionalities
+# and better understanding. For now, these models are good enough for
+# our upcoming prototype.
```

Comment:
This TODO is very vague. You can totally omit that or give a one liner to specify what exactly needs to be done.